src/common/LteCommon.cc

```diff
@@ -13,6 +13,14 @@
 #include "corenetwork/binder/LteBinder.h"
 #include "stack/mac/layer/LteMacEnb.h"
 #include "common/LteControlInfo.h"
+#include "inet/common/packet/dissector/ProtocolDissectorRegistry.h"
+#include "inet/networklayer/ipv4/Ipv4ProtocolDissector.h"
+
+using namespace inet;
+
+const inet::Protocol LteProtocol::lteuu("lteuu", "IPv4 (LTE Uu)");
+Register_Protocol_Dissector(&LteProtocol::lteuu, Ipv4ProtocolDissector);
```

Comment:
Ipv4ProtocolDissector doesn't seem right.

Reply:
Hmm, actually it is right since this is the IPv4 protocol on the Uu interface (we need a different protocol identifier than IPv4 for the traffic filter) - I renamed it to ipv4uu, so that it becomes more obvious that it is IPv4.